### DISCUSSION TITLE
861D637B-5562-8A01-734A-3D6FEFA68185

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#70] [General]
# 861D637B-5562-8A01-734A-3D6FEFA68185

New Horizons is a NASA/APL space probe that was launched as a part of NASA's New Frontiers program. It was launched in 2006 with the primary mission to perform a flyby study of the Pluto system in 2015, and a secondary mission to fly by and study one or more other Kuiper belt objects (KBOs) in the decade to follow, which became a mission to 486958 Arrokoth in 2019. It is the fifth space probe to achieve the escape velocity needed to leave the Solar System.

https://celestia.mobi/addon?item=861D637B-5562-8A01-734A-3D6FEFA68185

**Comment by galileokeplertycho** [comment]
The texture of this new horizon 3d model is very bad please bring the previous model from developer toutatis !!!!!!!!!
